templates/shared/actions/runner_edit.tmpl

```diff
@@ -16,9 +16,9 @@
 					<span>{{if .LastOnline}}{{TimeSinceUnix .LastOnline $.locale}}{{else}}{{$.locale.Tr "never"}}{{end}}</span>
 				</div>
 				<div class="field gt-dib gt-mr-4">
-					<label>{{.locale.Tr "actions.runners.agent_labels"}}</label>
+					<label>{{.locale.Tr "actions.runners.labels"}}</label>
```

Comment:
I noticed there are some translations which need to be updated or removed.

`actions.runners.labels`, `actions.runners.custom_labels` and `actions.runners.custom_labels_helper`